ods_ci/tests/Resources/Page/FeatureStore/FeatureStore.resource

```diff
@@ -6,9 +6,9 @@ Resource         ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
 
 
 *** Variables ***
-${FEATURE-STORE-RELEASE-TAG}             adjustment-release-v0.49.0
+${FEATURE-STORE-RELEASE-TAG}             v0.50.0
```

Comment:
We have 0.51 released last week, any reason we want to be on 0.50 ?

Reply:
Since RHOAI 2.23 supports feast released version 0.50 updating accordingly